raiden/__main__.py

```diff
@@ -5,7 +5,7 @@ def main():
     import gevent.monkey
     gevent.monkey.patch_all()
     from raiden.ui.cli import run
-    run()
+    run(auto_envvar_prefix='RAIDEN')
```

Comment:
Sorry, in short it enables all defined options to also be given via environment variables named `<PREFIX>_<UPPER_CAMELIZED_OPTION_NAME>`